### PR TITLE
updated http to https

### LINKED ISSUE #175: https in place of http
At least in the bottom link « This question is issued from http://dsp.stackexchange.com«.

### PR DESCRIPTION
**Pull request by Neilblaze**
#### Description
Updated `http` to `https`!

Fixes #175

### CODE REVIEW

File: sotoki/sotoki.py [hunk 1]
```diff
@@ -1652,7 +1652,7 @@ def run():
         url = domain
```
Comment (dattaz) [inline sotoki/sotoki.py:1652]: Here we can still have url assign to "http://" and not "https://".
I think the best way to process is to put the assignment to `url` outside of the if, with `url = "https://" + domain` after

**Review by dattaz — COMMENTED** [review]
Thanks you for your PR :)
I have add one comment, if you can have a look

File: sotoki/sotoki.py (hunk 1 above)
Comment (Neilblaze) [reply sotoki/sotoki.py:1652]: Ah I see. :thinking: That'll be a small fix. On it! :rocket:

Comment (Neilblaze) [reply sotoki/sotoki.py:1652]: @dattaz just a little bit of query, use this :point_down: or use `elif`? I'm a bit confused over here :sweat_smile: 

```
    if re.match("^https?://", url):
        url = domain
        domain = re.sub("^https?://", "", domain).split("/")[0]
    else:
        url = "http://" + domain
    publisher = arguments["<publisher>"]

    if re.match("^https?://", domain):
        url = domain
        domain = re.sub("^https?://", "", domain).split("/")[0]
    else:
        url = "https://" + domain
    publisher = arguments["<publisher>"]
```

Comment (dattaz) [reply sotoki/sotoki.py:1652]: `if re.match("^https?://", url):` check if url start with http:// or https:// to then extract the domain only.
I was thinking of :
`
    if re.match("^https?://", url):
        domain = re.sub("^https?://", "", domain).split("/")[0] 
    url = "https://" + domain
`

**Pull request closed without merging**